src/components/map/map.tsx

```diff
@@ -153,6 +198,15 @@ export class Map implements GxComponent {
       : MIN_ZOOM;
   }
 
+  private isSelectionMarkerSlot(): { exist: boolean; elem: Element } {
+    const slot = this.element.querySelector("[slot='selection-layer-marker']");
+    if (slot !== null) {
+      return { exist: true, elem: slot };
+    } else {
+      return { exist: false, elem: null };
+    }
+  }
+
```

Comment:
Could we simplify this function?
I feel that if the function just returns the slot, the caller can decide if it exists or not by checking if the function returns an object or null. In that case, I'd change the name to 
`getSelectionMarkerSlot`.

Reply:
I can't believe that... redundant code 🤦‍♂️ 
I will immediately fix that disrespect.

Reply:
Fixed! See changes in fd50e4d